cli/internal/terraform/terraform/gcp/modules/internal_loadbalancer/variables.tf

```diff
@@ -0,0 +1,50 @@
+variable "name" {
+  type        = string
+  description = "Base name of the load balancer."
+}
+
+variable "region" {
+  type        = string
+  description = "The region where the load balancer will be created."
+}
+
+variable "network" {
+  type        = string
+  description = "The network to which all network resources will be attached."
+}
+
+variable "backend_subnet" {
+  type        = string
+  description = "The subnet to which all backend network resources will be attached."
+}
+
+variable "health_check" {
+  type        = string
+  description = "The type of the health check. 'HTTPS' or 'TCP'."
```

Comment:
We could use an additional `validation.condition` check using `contains` if we want to restrict it to HTTPS & TCP.

See: 
* [validation](https://developer.hashicorp.com/terraform/language/expressions/custom-conditions#input-variable-validation)
* [contains](https://developer.hashicorp.com/terraform/language/functions/contains)